lib/cartopy/mpl/geoaxes.py

```diff
@@ -1276,14 +1276,29 @@ def imshow(self, img, *args, **kwargs):
             regrid_shape = self._regrid_shape_aspect(regrid_shape,
                                                      target_extent)
             warp_array = cartopy.img_transform.warp_array
+            unmodified_extent = extent
```

Comment:
Can't you still pass extent in down below in the alpha route and not add this new variable?

Reply:
I just realized extent is returned from the first warp_array call below 🤦Maybe change this to `original_extent`?

Reply:
No; "extent" is used as input in line 1281 and replaced as output in line 1279, and I haven't tracked where it's used in subsequent code.

Reply:
OK---will change to `original_extent`.